reactjs-films-homework/src/languages/getLanguage.ts

```diff
@@ -0,0 +1,26 @@
+import data from './languagesData';
+
+const lang = (language: string) => {
+  let globalLang;
+
+  switch (language) {
+    case '':
```

Comment:
Why do you need this case? You should set up your app the way which allows only 2 cases 'EN' and 'RU'. Shouldn't be the case when lang is empty. Also, you can set up one of the languages as a default value.